src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -676,44 +673,44 @@ private static void PrintLatencies(List<Worker> workers, boolean outputVerboseRe
   private static void PrintQueryAttempts(List<Worker> workers, WorkloadConfiguration workConf) {
     int numTxnTypes = workConf.getNumTxnTypes();
     int numTriesPerProc = workConf.getMaxRetriesPerTransaction() + 1;
-    int[][] totalRetries = new int[numTxnTypes][numTriesPerProc];
-    int[] totalFailures = new int[numTxnTypes];
+    int[][] totalFailedTries = new int[numTxnTypes][numTriesPerProc];
     for (Worker w : workers) {
       for (int i = 0; i < numTxnTypes; ++i) {
-        totalFailures[i] += w.getTotalFailures()[i];
         for (int tryIdx = 0; tryIdx < numTriesPerProc; ++tryIdx) {
-          totalRetries[i][tryIdx] += w.getTotalTries()[i][tryIdx];
+          totalFailedTries[i][tryIdx] += w.getTotalFailedTries()[i][tryIdx];
         }
       }
     }
 
     StringBuilder resultOut = new StringBuilder();
 
     resultOut.append("\n");
-    resultOut.append("========RETRIES (DEPRECATED STATS)===========\n");
-    resultOut.append("   Operation   | Total Created |");
-    for (int i = 1; i < numTriesPerProc; ++i) {
-      resultOut.append(String.format(" Retry Number %1d |", i));
+    resultOut.append("=================== RETRY ATTEMPTS ====================\n");
+    resultOut.append("  Transaction  |    Count  |");
+    for (int i = 0; i < numTriesPerProc; ++i) {
+      resultOut.append(String.format(" Retry #%1d - Failure Count |", i));
```

Comment:
maybe use `i + 1` so we start with "Retry #1" in the output stats

Reply:
Retry #1 can be interpreted as a retry however the first run is not a retry. It is the first try. Hence kept it as Retry #0